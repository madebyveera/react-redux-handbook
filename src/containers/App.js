import React from 'react'
import Footer from './Footer'
import AddTodo from '../components/AddTodo'
import VisibleTodoList from './VisibleTodoList'
import Navbar from './../components/navbar'

const App = () => (
  <>
    <header>
        <Navbar />
    </header>
    <main>
        <section>
            <div className="container-fluid">
                <div className="row">
                    <div className="offset-3 col-6">
                        <AddTodo />
                        <VisibleTodoList />
                        <Footer />
                    </div>
                </div>
            </div>
        </section>
    </main>
  </>
)

export default App
