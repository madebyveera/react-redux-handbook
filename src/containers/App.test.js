import React from "react";
import { Provider } from "react-redux";
import renderer from 'react-test-renderer';
import App from "./App";
import store from "./../store/configureStore";

describe('My Connected React-Redux Component', () => {
  let component;
  beforeEach(() => {
    component = renderer.create(
      <Provider store={store}>
        <App />
      </Provider>
    );
  });

  it("renders learn react link", () => {
    expect(component.toJSON()).toMatchSnapshot();
  });
});