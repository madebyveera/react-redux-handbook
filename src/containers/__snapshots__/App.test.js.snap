// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`My Connected React-Redux Component renders learn react link 1`] = `
Array [
  <header>
    <nav
      className="navbar navbar-expand-lg navbar-light bg-light"
    >
      <a
        className="navbar-brand"
        href="#/"
      >
        <label>
          Todo
        </label>
      </a>
      <button
        aria-controls="navbarSupportedContent"
        aria-expanded="false"
        aria-label="Toggle navigation"
        className="navbar-toggler"
        data-target="#navbarSupportedContent"
        data-toggle="collapse"
        type="button"
      >
        <span
          className="navbar-toggler-icon"
        />
      </button>
    </nav>
  </header>,
  <main>
    <section>
      <div
        className="container-fluid"
      >
        <div
          className="row"
        >
          <div
            className="offset-3 col-6"
          >
            <div
              className="container"
            >
              <form>
                <div
                  className="form-group m-4"
                >
                  <div
                    className="input-group"
                  >
                    <input
                      className="form-control"
                      onChange={[Function]}
                      placeholder="Add Todo"
                      value="Use Redux"
                    />
                    <div
                      className="input-group-append"
                    >
                      <button
                        className="btn btn-primary"
                        onClick={[Function]}
                        type="button"
                      >
                        Add Todo
                      </button>
                    </div>
                  </div>
                </div>
              </form>
            </div>
            <table
              className="table"
            >
              <thead>
                <tr>
                  <th>
                    #
                  </th>
                  <th>
                    Entity
                  </th>
                </tr>
              </thead>
              <tbody />
            </table>
            <div
              className="d-flex justify-content-end"
            >
              <div
                className="btn-group"
              >
                <button
                  className="btn btn-primary"
                  disabled={true}
                  onClick={[Function]}
                  style={
                    Object {
                      "marginLeft": "4px",
                    }
                  }
                  type="button"
                >
                  All
                </button>
                <button
                  className="btn btn-warning"
                  disabled={false}
                  onClick={[Function]}
                  style={
                    Object {
                      "marginLeft": "4px",
                    }
                  }
                  type="button"
                >
                  Active
                </button>
                <button
                  className="btn btn-success"
                  disabled={false}
                  onClick={[Function]}
                  style={
                    Object {
                      "marginLeft": "4px",
                    }
                  }
                  type="button"
                >
                  Completed
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </section>
  </main>,
]
`;
